spec/text_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'text.rb'
+require 'twilio-ruby'
+
+describe TextConfirm do
+  it 'initialises with @from with an environmental variable' do
+    expect(subject.from).to eq ENV['SECRET_TWIL_NUM']
+  end
+
+  it 'initialises with @to with an environmental variable' do
+    expect(subject.to).to eq ENV['SECRET_MOBILE']
+  end
+
```

Comment:
Nice - I couldn't think how to test this part of my code. Seeing your's makes perfect sense!